hackathon_site/event/jinja2/event/landing.html

```diff
@@ -27,292 +29,296 @@
-                                participate! We are all here to learn :)
-                            </span>
-                        </div>
-                    </li>
-                    <li>
-                        <div class="collapsible-header">
-                            What equipment and support would be provided?
-                        </div>
-                        <div class="collapsible-body">
-                            <span>
-                                Unfortunately, due to the current situation of COVID19. We won’t be able to provide any hardware. However, there would be lots of mentors ready to help you! During the
-                                online hackathon, there would be a breakout room for all the mentors where you would be able to access at all times and get immediate help.
-                            </span>
-                        </div>
-                    </li>
-                    <li>
-                        <div class="collapsible-header">
-                            How do online hackathons work?
-                        </div>
-                        <div class="collapsible-body">
-                            <span>
-                                The entire hackathon would be conducted online. You would be able to stay safe and comfortably at home while doing this hackathon. You would be able to communicate to
-                                your team via online meetings and you would be submitting your final products online.
-                            </span>
-                        </div>
-                    </li>
-                </ul>
+                <div class="col s12 m6">
+                    <ul class="collapsible">
+                        <li>
+                            <div class="collapsible-header">
+                                Who can apply? What if I have no experience?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>
+                                    Applicants that are currently in either highschool or university are welcomed to apply! We are a beginner friendly hackathon, so no hackathon experience is needed to participate! We
+                                    are all here to learn :)
+                                </span>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="collapsible-header">
+                                What equipment and support would be provided?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>
+                                    Unfortunately, due to the current situation of COVID19. We won’t be able to provide any hardware. However, there would be lots of mentors ready to help you! During the online
```

Comment:
Should be a comma after COVID19, not a fullstop.

The use of the word "would" implies that that would have been the case, but won't. Should it be replaced with "will"?